src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
-package core.basesyntax;
-
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
-}
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomIndex = new Random();

```

Comment:
you may declare this field as `final` since you don't change it after initialization